ccmlib/scylla_node.py

```diff
@@ -609,7 +609,10 @@ def process_opts(opts):
             elif k != '--memory':
                 args.append(k)
                 if v:
-                    args.append(v)
+                    args.append(v[0])
+                    for val in v[1:]:
+                        args.append(k)
+                        args.append(val)
```

Comment:
if we really want to unify the argument to be multiple values, i'd suggest:
```py
elif k != '--memory':
    for val in v:
        args += [k, val]
```

Reply:
yeah that a bit clearer

Reply:
used that, and it's much better, thanks.